### DISCUSSION TITLE
Scale min/max color ramp to map bbox

### DISCUSSION THREAD

**Discussion by gagecarto** [developmentseed/titiler#475] [Q&A] [answered]
I would like the min and max of my color ramp to respond to the tiles within the map extent. I was thinking I could request stats for the current map extent and then use those min/max in my tile requests. Any other suggestions?

Is it possible to request cog/statistics with a bbox parameter?

**Comment by vincentsarago** [comment, marked as answer]
@gagecarto there is a [POST] `/statistics` endpoint where you can get stats for a geojson feature.

**Reply by gagecarto** [reply]
@vincentsarago - This is great.. 

What I am doing at this point is triggering a call to `[POST]/statistics` when the map stops moving. I then get the min/max and rebuild the tile-json `rescale=` parameter. 

It's not perfect since I need to wait for the [POST] response but it's really cool. I am so impressed with how quick the stats area, especially with the binning etc.
